firebase-firestore/src/main/java/com/google/firebase/firestore/local/SQLiteIndexManager.java

```diff
@@ -324,7 +324,7 @@ public IndexOffset getMinOffset(Target target) {
     } else {
       // There is an implicit AND operation between all the filters stored in the target.
       List<Filter> dnf =
-          LogicUtils.DnfTransform(
+          LogicUtils.getDnfTerms(
```

Comment:
This should be `getDnfTerms` per the style guide.

Reply:
Done